LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2017, Voxel51, LLC
+Copyright (c) 2017, KUNYI
```

Comment:
lol! you decided to claim copyrights for the entire ETA library now?

Please revert this

Reply:
Sorry for that. I did not realize it. solved